care/facility/static_data/icd11.py

```diff
@@ -34,18 +34,15 @@ def get_representation(self) -> ICD11Object:
 def load_icd11_diagnosis():
     print("Loading ICD11 Diagnosis into the redis cache...", end="", flush=True)
 
-    icd_objs = ICD11Diagnosis.objects.order_by("id").values_list(
-        "id", "label", "meta_chapter_short"
-    )
-    paginator = Paginator(icd_objs, 5000)
-    for page_number in paginator.page_range:
-        for diagnosis in paginator.page(page_number).object_list:
-            ICD11(
-                id=diagnosis[0],
-                label=diagnosis[1],
-                chapter=diagnosis[2] or "",
-                has_code=1 if re.match(DISEASE_CODE_PATTERN, diagnosis[1]) else 0,
-            ).save()
+    with open("data/icd11.json", "r") as f:
+        icd_data = json.load(f)
+    for diagnosis in icd_data:
+        ICD11(
+            id=icd11_id_to_int(diagnosis.get("ID")),
+            label=diagnosis.get("label"),
+            chapter=diagnosis.get("meta_chapter_short", ""),
```

Comment:
this is a generated field,
https://github.com/DraKen0009/care/blob/09813dfaf1e05ab76dc833bd135734ca01c0fa45/care/facility/management/commands/load_icd11_diagnoses_data.py#L142-L143